test.py

```diff
@@ -1546,6 +1552,7 @@ async def reap(done, pending, signaled):
         return
     finally:
         await TestSuite.artifacts.cleanup_before_exit()
+        mock_s3_server.stop()
```

Comment:
It should do `TestSuite.artifacts.add_exit_artifact(None, mock_s3_server.stop)` instead, like minio server does

Reply:
Done